fn/chocolatey/functions.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>functions</id>
+    <version>0.2.25</version>
+    <packageSourceUrl>https://github.com/iron-io/functions/</packageSourceUrl>
+    
+    <title>Iron.io Functions (Install)</title>
+    <authors>Iron.io Inc.</authors>
+    <projectUrl>https://github.com/iron-io/functions/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/functions.png</iconUrl>-->
+    <copyright>2017 Functions Iron.io</copyright>
+    <tags>serverless faas docker lambda</tags>
```

Comment:
I'd say keep serverless and add functions. We do want to be associated with the term serverless as that's what everyone is referring to it as for the time being.